steppers/ldo/42sth40-1684ac/2130/24v-0.4a-z.cfg

```diff
@@ -1,6 +1,7 @@
 [tmc2130 stepper_z]
 sense_resistor: 0.30
 run_current: 0.4
+hold_current: 0.2
```

Comment:
Delete this (see: https://www.klipper3d.org/TMC_Drivers.html#prefer-to-not-specify-a-hold_current)

Reply:
Done